packages/queued-request-controller/CHANGELOG.md

```diff
@@ -6,4 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/core/
+## [0.1.0]
+### Added
+- Initial release
+
+[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/queued-request-controller@0.1.0...HEAD
+[0.1.0]: https://github.com/MetaMask/core/compare/@metamask/queued-request-controller@0.1.0
```

Comment:
The URL for the initial release points to a tag and not a comparison view (this should fix CI):

``` suggestion
[0.1.0]: https://github.com/MetaMask/core/releases/tag/@metamask/queued-request-controller@0.1.0